tasks.py

```diff
@@ -40,7 +40,10 @@ def tests(c):
     """Run the redis-py test suite against the current python,
     with and without hiredis.
     """
+    print("Starting Redis tests")
     run("tox -e plain -e hiredis")
+    print("Starting RedisCluster tests")
+    run("tox -e plain -e hiredis -- --redis-url=redis://localhost:16379/0")
```

Comment:
Can we allow for an environment override. Currently we allow the override in pyenv, and tox - I'm thinking something like:

```
    REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:16379/0")
    run("tox -e plain -e hiredis -- --redis-url={}".format(REDIS_URL)
```